Build failure: roslyn-ls

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

Build roslyn-ls

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

_No response_

### Relevant log output

```console
nix build .#roslyn-ls
this derivation will be built:
  /nix/store/v6isk03d7vmwc5cv0v8fn6qckcyhxnv7-roslyn-ls-5.0.0-2.25371.17.drv
building '/nix/store/v6isk03d7vmwc5cv0v8fn6qckcyhxnv7-roslyn-ls-5.0.0-2.25371.17.drv'...
roslyn-ls> Running phase: unpackPhase
roslyn-ls> unpacking source archive /nix/store/zc3kvmqrw0k9jysn5hr5x4rwigx72m6d-source
roslyn-ls> source root is source
roslyn-ls> Running phase: patchPhase
roslyn-ls> applying patch /nix/store/l4gq3hpqn7vaqp0vb5csgy4lsl6v1kxm-force-sdk_8_0.patch
roslyn-ls> patching file eng/targets/TargetFrameworks.props
roslyn-ls> Hunk #1 succeeded at 17 with fuzz 2.
roslyn-ls> patching file src/Workspaces/MSBuild/BuildHost/Microsoft.CodeAnalysis.Workspaces.MSBuild.BuildHost.csproj
roslyn-ls> Hunk #1 succeeded at 30 with fuzz 2 (offset 2 lines).
roslyn-ls> Hunk #2 succeeded at 44 (offset 2 lines).
roslyn-ls> applying patch /nix/store/k082axz1c0bylyvfrk2jz99wcfahqjq5-cachedirectory.patch
roslyn-ls> patching file src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Program.cs
roslyn-ls> Hunk #1 succeeded at 94 with fuzz 2 (offset -3 lines).
roslyn-ls> Running phase: configureNuget
roslyn-ls> Running phase: updateAutotoolsGnuConfigScriptsPhase
roslyn-ls> Running phase: configurePhase
roslyn-ls> Executing dotnetConfigureHook
roslyn-ls> Tool 'dotnet-format' (version '7.0.360304') was restored. Available commands: dotnet-format
roslyn-ls> Tool 'powershell' (version '7.0.0') was restored. Available commands: pwsh
roslyn-ls>
roslyn-ls> Restore was successful.
roslyn-ls>
roslyn-ls>   Determining projects to restore...
roslyn-ls> /build/source/src/Tools/ExternalAccess/Razor/Features/Microsoft.CodeAnalysis.ExternalAccess.Razor.Features.csproj : error NU1101: Unable to find package System.Diagnostics.Tracing. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
roslyn-ls> /build/source/src/Workspaces/Remote/Core/Microsoft.CodeAnalysis.Remote.Workspaces.csproj : error NU1101: Unable to find package System.Diagnostics.Tracing. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
roslyn-ls> /build/source/src/Workspaces/Remote/Core/Microsoft.CodeAnalysis.Remote.Workspaces.csproj : error NU1101: Unable to find package System.Diagnostics.Process. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
roslyn-ls> /build/source/src/Workspaces/Remote/Core/Microsoft.CodeAnalysis.Remote.Workspaces.csproj : error NU1101: Unable to find package System.Diagnostics.TraceSource. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
roslyn-ls> /build/source/src/LanguageServer/ExternalAccess/VisualDiagnostics/Microsoft.CodeAnalysis.ExternalAccess.VisualDiagnostics.csproj : error NU1101: Unable to find package System.Diagnostics.Tracing. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
roslyn-ls> /build/source/src/Workspaces/Remote/Core/Microsoft.CodeAnalysis.Remote.Workspaces.csproj : error NU1101: Unable to find package System.IO.Pipes. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
roslyn-ls> /build/source/src/LanguageServer/ExternalAccess/VisualDiagnostics/Microsoft.CodeAnalysis.ExternalAccess.VisualDiagnostics.csproj : error NU1101: Unable to find package System.Diagnostics.Process. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
roslyn-ls> /build/source/src/LanguageServer/ExternalAccess/VisualDiagnostics/Microsoft.CodeAnalysis.ExternalAccess.VisualDiagnostics.csproj : error NU1101: Unable to find package System.Diagnostics.TraceSource. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
roslyn-ls> /build/source/src/Tools/ExternalAccess/Razor/Features/Microsoft.CodeAnalysis.ExternalAccess.Razor.Features.csproj : error NU1101: Unable to find package System.Diagnostics.Process. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
roslyn-ls> /build/source/src/LanguageServer/ExternalAccess/VisualDiagnostics/Microsoft.CodeAnalysis.ExternalAccess.VisualDiagnostics.csproj : error NU1101: Unable to find package System.IO.Pipes. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
roslyn-ls> /build/source/src/Tools/ExternalAccess/Razor/Features/Microsoft.CodeAnalysis.ExternalAccess.Razor.Features.csproj : error NU1101: Unable to find package System.Diagnostics.TraceSource. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
roslyn-ls> /build/source/src/Tools/ExternalAccess/Razor/Features/Microsoft.CodeAnalysis.ExternalAccess.Razor.Features.csproj : error NU1101: Unable to find package System.IO.Pipes. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
roslyn-ls>   Restored /build/source/src/Workspaces/MSBuild/BuildHost/Microsoft.CodeAnalysis.Workspaces.MSBuild.BuildHost.csproj (in 368 ms).
roslyn-ls>   Restored /build/source/src/Compilers/Core/Portable/Microsoft.CodeAnalysis.csproj (in 382 ms).
roslyn-ls>   Restored /build/source/src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/CSharpSyntaxGenerator.csproj (in 348 ms).
roslyn-ls>   Restored /build/source/src/Compilers/CSharp/Portable/Microsoft.CodeAnalysis.CSharp.csproj (in 381 ms).
roslyn-ls>   Restored /build/source/src/Tools/ExternalAccess/RazorCompiler/Microsoft.CodeAnalysis.ExternalAccess.RazorCompiler.csproj (in 380 ms).
roslyn-ls>   Restored /build/source/src/Features/ExternalAccess/Copilot/Microsoft.CodeAnalysis.ExternalAccess.Copilot.csproj (in 376 ms).
roslyn-ls>   Restored /build/source/src/Scripting/Core/Microsoft.CodeAnalysis.Scripting.csproj (in 382 ms).
roslyn-ls>   Restored /build/source/src/Tools/ExternalAccess/Xaml/Microsoft.CodeAnalysis.ExternalAccess.Xaml.csproj (in 371 ms).
roslyn-ls>   Restored /build/source/src/LanguageServer/ExternalAccess/Copilot/Microsoft.CodeAnalysis.LanguageServer.ExternalAccess.Copilot.csproj (in 371 ms).
roslyn-ls>   Restored /build/source/src/LanguageServer/ExternalAccess/CompilerDeveloperSDK/Microsoft.CodeAnalysis.ExternalAccess.CompilerDeveloperSDK.csproj (in 348 ms).
roslyn-ls>   Restored /build/source/src/Workspaces/CSharp/Portable/Microsoft.CodeAnalysis.CSharp.Workspaces.csproj (in 389 ms).
roslyn-ls>   Restored /build/source/src/Features/ExternalAccess/AspNetCore/Microsoft.CodeAnalysis.ExternalAccess.AspNetCore.csproj (in 348 ms).
roslyn-ls>   Restored /build/source/src/Workspaces/Core/Portable/Microsoft.CodeAnalysis.Workspaces.csproj (in 399 ms).
roslyn-ls>   Restored /build/source/src/Features/Core/Portable/Microsoft.CodeAnalysis.Features.csproj (in 404 ms).
roslyn-ls>   Restored /build/source/src/Features/CSharp/Portable/Microsoft.CodeAnalysis.CSharp.Features.csproj (in 403 ms).
roslyn-ls>   Restored /build/source/src/Workspaces/MSBuild/Core/Microsoft.CodeAnalysis.Workspaces.MSBuild.csproj (in 415 ms).
roslyn-ls>   Restored /build/source/src/LanguageServer/Protocol/Microsoft.CodeAnalysis.LanguageServer.Protocol.csproj (in 413 ms).
roslyn-ls>   Restored /build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj (in 405 ms).
roslyn-ls>   Failed to restore /build/source/src/LanguageServer/ExternalAccess/VisualDiagnostics/Microsoft.CodeAnalysis.ExternalAccess.VisualDiagnostics.csproj (in 389 ms).
roslyn-ls>   Failed to restore /build/source/src/Workspaces/Remote/Core/Microsoft.CodeAnalysis.Remote.Workspaces.csproj (in 426 ms).
roslyn-ls>   Failed to restore /build/source/src/Tools/ExternalAccess/Razor/Features/Microsoft.CodeAnalysis.ExternalAccess.Razor.Features.csproj (in 445 ms).
error: Cannot build '/nix/store/v6isk03d7vmwc5cv0v8fn6qckcyhxnv7-roslyn-ls-5.0.0-2.25371.17.drv'.
       Reason: builder failed with exit code 1.
       Output paths:
         /nix/store/fq8d8nrppah1anicjx9z49m4gazvlz7r-roslyn-ls-5.0.0-2.25371.17
       Last 50 log lines:
       > Hunk #1 succeeded at 17 with fuzz 2.
       > patching file src/Workspaces/MSBuild/BuildHost/Microsoft.CodeAnalysis.Workspaces.MSBuild.BuildHost.csproj
       > Hunk #1 succeeded at 30 with fuzz 2 (offset 2 lines).
       > Hunk #2 succeeded at 44 (offset 2 lines).
       > applying patch /nix/store/k082axz1c0bylyvfrk2jz99wcfahqjq5-cachedirectory.patch
       > patching file src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Program.cs
       > Hunk #1 succeeded at 94 with fuzz 2 (offset -3 lines).
       > Running phase: configureNuget
       > Running phase: updateAutotoolsGnuConfigScriptsPhase
       > Running phase: configurePhase
       > Executing dotnetConfigureHook
       > Tool 'dotnet-format' (version '7.0.360304') was restored. Available commands: dotnet-format
       > Tool 'powershell' (version '7.0.0') was restored. Available commands: pwsh
       >
       > Restore was successful.
       >
       >   Determining projects to restore...
       > /build/source/src/Tools/ExternalAccess/Razor/Features/Microsoft.CodeAnalysis.ExternalAccess.Razor.Features.csproj : error NU1101: Unable to find package System.Diagnostics.Tracing. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
       > /build/source/src/Workspaces/Remote/Core/Microsoft.CodeAnalysis.Remote.Workspaces.csproj : error NU1101: Unable to find package System.Diagnostics.Tracing. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
       > /build/source/src/Workspaces/Remote/Core/Microsoft.CodeAnalysis.Remote.Workspaces.csproj : error NU1101: Unable to find package System.Diagnostics.Process. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
       > /build/source/src/Workspaces/Remote/Core/Microsoft.CodeAnalysis.Remote.Workspaces.csproj : error NU1101: Unable to find package System.Diagnostics.TraceSource. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
       > /build/source/src/LanguageServer/ExternalAccess/VisualDiagnostics/Microsoft.CodeAnalysis.ExternalAccess.VisualDiagnostics.csproj : error NU1101: Unable to find package System.Diagnostics.Tracing. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
       > /build/source/src/Workspaces/Remote/Core/Microsoft.CodeAnalysis.Remote.Workspaces.csproj : error NU1101: Unable to find package System.IO.Pipes. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
       > /build/source/src/LanguageServer/ExternalAccess/VisualDiagnostics/Microsoft.CodeAnalysis.ExternalAccess.VisualDiagnostics.csproj : error NU1101: Unable to find package System.Diagnostics.Process. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
       > /build/source/src/LanguageServer/ExternalAccess/VisualDiagnostics/Microsoft.CodeAnalysis.ExternalAccess.VisualDiagnostics.csproj : error NU1101: Unable to find package System.Diagnostics.TraceSource. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
       > /build/source/src/Tools/ExternalAccess/Razor/Features/Microsoft.CodeAnalysis.ExternalAccess.Razor.Features.csproj : error NU1101: Unable to find package System.Diagnostics.Process. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
       > /build/source/src/LanguageServer/ExternalAccess/VisualDiagnostics/Microsoft.CodeAnalysis.ExternalAccess.VisualDiagnostics.csproj : error NU1101: Unable to find package System.IO.Pipes. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
       > /build/source/src/Tools/ExternalAccess/Razor/Features/Microsoft.CodeAnalysis.ExternalAccess.Razor.Features.csproj : error NU1101: Unable to find package System.Diagnostics.TraceSource. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
       > /build/source/src/Tools/ExternalAccess/Razor/Features/Microsoft.CodeAnalysis.ExternalAccess.Razor.Features.csproj : error NU1101: Unable to find package System.IO.Pipes. No packages exist with this id in source(s): /nix/store/fqwd90li0rgllz17rjl8s7ymf9kzhr6i-dotnet-sdk-10.0.100-preview.7.25380.108/share/dotnet/library-packs, _nix [/build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj]
       >   Restored /build/source/src/Workspaces/MSBuild/BuildHost/Microsoft.CodeAnalysis.Workspaces.MSBuild.BuildHost.csproj (in 368 ms).
       >   Restored /build/source/src/Compilers/Core/Portable/Microsoft.CodeAnalysis.csproj (in 382 ms).
       >   Restored /build/source/src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/CSharpSyntaxGenerator.csproj (in 348 ms).
       >   Restored /build/source/src/Compilers/CSharp/Portable/Microsoft.CodeAnalysis.CSharp.csproj (in 381 ms).
       >   Restored /build/source/src/Tools/ExternalAccess/RazorCompiler/Microsoft.CodeAnalysis.ExternalAccess.RazorCompiler.csproj (in 380 ms).
       >   Restored /build/source/src/Features/ExternalAccess/Copilot/Microsoft.CodeAnalysis.ExternalAccess.Copilot.csproj (in 376 ms).
       >   Restored /build/source/src/Scripting/Core/Microsoft.CodeAnalysis.Scripting.csproj (in 382 ms).
       >   Restored /build/source/src/Tools/ExternalAccess/Xaml/Microsoft.CodeAnalysis.ExternalAccess.Xaml.csproj (in 371 ms).
       >   Restored /build/source/src/LanguageServer/ExternalAccess/Copilot/Microsoft.CodeAnalysis.LanguageServer.ExternalAccess.Copilot.csproj (in 371 ms).
       >   Restored /build/source/src/LanguageServer/ExternalAccess/CompilerDeveloperSDK/Microsoft.CodeAnalysis.ExternalAccess.CompilerDeveloperSDK.csproj (in 348 ms).
       >   Restored /build/source/src/Workspaces/CSharp/Portable/Microsoft.CodeAnalysis.CSharp.Workspaces.csproj (in 389 ms).
       >   Restored /build/source/src/Features/ExternalAccess/AspNetCore/Microsoft.CodeAnalysis.ExternalAccess.AspNetCore.csproj (in 348 ms).
       >   Restored /build/source/src/Workspaces/Core/Portable/Microsoft.CodeAnalysis.Workspaces.csproj (in 399 ms).
       >   Restored /build/source/src/Features/Core/Portable/Microsoft.CodeAnalysis.Features.csproj (in 404 ms).
       >   Restored /build/source/src/Features/CSharp/Portable/Microsoft.CodeAnalysis.CSharp.Features.csproj (in 403 ms).
       >   Restored /build/source/src/Workspaces/MSBuild/Core/Microsoft.CodeAnalysis.Workspaces.MSBuild.csproj (in 415 ms).
       >   Restored /build/source/src/LanguageServer/Protocol/Microsoft.CodeAnalysis.LanguageServer.Protocol.csproj (in 413 ms).
       >   Restored /build/source/src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj (in 405 ms).
       >   Failed to restore /build/source/src/LanguageServer/ExternalAccess/VisualDiagnostics/Microsoft.CodeAnalysis.ExternalAccess.VisualDiagnostics.csproj (in 389 ms).
       >   Failed to restore /build/source/src/Workspaces/Remote/Core/Microsoft.CodeAnalysis.Remote.Workspaces.csproj (in 426 ms).
       >   Failed to restore /build/source/src/Tools/ExternalAccess/Razor/Features/Microsoft.CodeAnalysis.ExternalAccess.Razor.Features.csproj (in 445 ms).
       For full logs, run:
         nix log /nix/store/v6isk03d7vmwc5cv0v8fn6qckcyhxnv7-roslyn-ls-5.0.0-2.25371.17.drv
```

### Additional context

_No response_

### System metadata


 - system: `"x86_64-linux"`
 - host os: `Linux 6.16.1, NixOS, 25.11 (Xantusia), 25.11.20250819.2007595`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.30.2`
 - channels(root): `""`
 - nixpkgs: `/nix/store/pxnx7dbr0pz3qpgw2r315wdbb00afdw7-source`

### Notify maintainers

@konradmalik 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/ro/roslyn-ls/package.nix
+++ b/pkgs/by-name/ro/roslyn-ls/package.nix
@@ -53,18 +53,18 @@ in
 buildDotnetModule (finalAttrs: rec {
   inherit pname dotnet-sdk dotnet-runtime;
 
-  vsVersion = "2.87.26";
+  vsVersion = "2.90.51-prerelease";
   src = fetchFromGitHub {
     owner = "dotnet";
     repo = "roslyn";
     rev = "VSCode-CSharp-${vsVersion}";
-    hash = "sha256-5XDE2fwBga1hhXgaNG46vruljnKulLR7yIT5BLjJBGA=";
+    hash = "sha256-l2/EIvN/GFIyCZRNBnS7bAzkYB1wZbD1DxD1EW040X4=";
   };
 
   # versioned independently from vscode-csharp
   # "roslyn" in here:
   # https://github.com/dotnet/vscode-csharp/blob/main/package.json
-  version = "5.0.0-2.25371.17";
+  version = "5.0.0-2.25424.1";
   projectFile = "src/LanguageServer/${project}/${project}.csproj";
   useDotnetFromEnv = true;
   nugetDeps = ./deps.json;

